cmd/microcloud/status_test.go

```diff
@@ -0,0 +1,492 @@
+					},
+				},
+			},
+			expectedWarnings: []Warning{
+				{Level: Error, Message: "Reliability risk: 3 systems are required for effective fault tolerance"},
+				{Level: Warn, Message: "MicroOVN is not installed on micro01"},
+				{Level: Warn, Message: "MicroCeph is not installed on micro01"},
+			},
+			expectedStatus: map[string]microTypes.MemberStatus{"micro01": microTypes.MemberOnline, "micro02": microTypes.MemberOnline},
+		},
+		{
+			desc: "2 node MicroCloud with LXD",
+			statuses: []types.Status{
+				{
+					Name:    "micro01",
+					Address: "10.0.0.101",
+					Clusters: map[types.ServiceType][]microTypes.ClusterMember{
+						types.MicroCloud: {genMember("micro01", microTypes.MemberOnline), genMember("micro02", microTypes.MemberOnline)},
+						types.LXD:        {genMember("micro01", microTypes.MemberOnline), genMember("micro02", microTypes.MemberOnline)},
+					},
+				},
+				{
+					Name:    "micro02",
+					Address: "10.0.0.102",
+					Clusters: map[types.ServiceType][]microTypes.ClusterMember{
+						types.MicroCloud: {genMember("micro01", microTypes.MemberOnline), genMember("micro02", microTypes.MemberOnline)},
+						types.LXD:        {genMember("micro01", microTypes.MemberOnline), genMember("micro02", microTypes.MemberOnline)},
+					},
+				},
+			},
+			expectedWarnings: []Warning{
+				{Level: Error, Message: "Reliability risk: 3 systems are required for effective fault tolerance"},
+				{Level: Warn, Message: "MicroOVN is not installed on micro01, micro02"},
+				{Level: Warn, Message: "MicroCeph is not installed on micro01, micro02"},
+			},
+			expectedStatus: map[string]microTypes.MemberStatus{"micro01": microTypes.MemberOnline, "micro02": microTypes.MemberOnline},
+		},
+		{
+			desc: "2 node MicroCloud with LXD, member is not online",
+			statuses: []types.Status{
+				{
+					Name:    "micro01",
+					Address: "10.0.0.101",
+					Clusters: map[types.ServiceType][]microTypes.ClusterMember{
+						types.MicroCloud: {genMember("micro01", microTypes.MemberOnline), genMember("micro02", microTypes.MemberOnline)},
+						types.LXD:        {genMember("micro01", microTypes.MemberOnline), genMember("micro02", "some unknown status")},
+					},
+				},
+				{
+					Name:    "micro02",
+					Address: "10.0.0.102",
+					Clusters: map[types.ServiceType][]microTypes.ClusterMember{
+						types.MicroCloud: {genMember("micro01", microTypes.MemberOnline), genMember("micro02", microTypes.MemberOnline)},
+						types.LXD:        {genMember("micro01", microTypes.MemberOnline), genMember("micro02", microTypes.MemberOnline)},
+					},
+				},
+			},
+			expectedWarnings: []Warning{
+				{Level: Error, Message: "Reliability risk: 3 systems are required for effective fault tolerance"},
+				{Level: Warn, Message: "MicroOVN is not installed on micro01, micro02"},
+				{Level: Warn, Message: "MicroCeph is not installed on micro01, micro02"},
+				{Level: Warn, Message: "micro02 is not available"},
```

Comment:
If a cluster member is offline, shouldn't this be an error?

Reply:
I've made these into errors, and now they also include a comma delimited list of services which are not available.